<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckCutomDialog</class>
 <widget class="QDialog" name="CheckCutomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>来客信息登记表</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>630</width>
           <height>300</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEditCustomName">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>客户姓名</string>
                </property>
               </widget>
              </item>
              <item row="3" column="4" colspan="2">
               <widget class="QLineEdit" name="lineEditCustomCash">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>客户押金</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>*入住时间:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2" colspan="4">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>从：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateEditFrom">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::UpDownArrows</enum>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>到：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateEditTo">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="lineEditRoomId">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="placeholderText">
                   <string>请输入房间编号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbnFindId">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>OpenHandCursor</cursorShape>
                  </property>
                  <property name="toolTip">
                   <string>查找房间编号</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="image.qrc">
                    <normaloff>:/image/find.png</normaloff>:/image/find.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>共</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDay">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="placeholderText">
                   <string>天数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>天</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>性别：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="3">
               <widget class="QLabel" name="label_13">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>订单号：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLineEdit" name="lineEditroomTypeId">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="placeholderText">
                   <string>请输入房间类型编号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbnFindTypeId">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>OpenHandCursor</cursorShape>
                  </property>
                  <property name="toolTip">
                   <string>查找房间类型编号</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="image.qrc">
                    <normaloff>:/image/find.png</normaloff>:/image/find.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>*客户姓名：</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_12">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>*身份证：</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEditCustomPhone">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>客户电话</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEditCustomIcard">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>身份证号</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>*房间编号:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_11">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>客户电话：</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="label_16">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>客户押金：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3" colspan="2">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>*房间类型编号:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="4" colspan="2">
               <widget class="QLineEdit" name="lineEditCustomOrderId">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>BlankCursor</cursorShape>
                </property>
                <property name="placeholderText">
                 <string>自动生成</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="5">
               <widget class="QLineEdit" name="lineEditRemark">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>相应的备注信息</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_14">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>备注：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QComboBox" name="comboBoxSex">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>男</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>女</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pbnCheckOk">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="toolTip">
                 <string>登记</string>
                </property>
                <property name="text">
                 <string>登记</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
